src/pages/ReferencePage.vue

```diff
@@ -1,162 +1,276 @@
 <template>
   <div>
-    <b-container
-      fluid
-      class="p-4 bg-light"
-    >
-      <b-row>
-        <div class="col-12">
-          <h1> Unsupervised Learning </h1>
-          <p> 
+    <div> 
```

Comment:
add the class `class="mx-2"` to the tag so that the wordings can be aligned to those below